Compiler/General.py

```diff
@@ -110,32 +114,41 @@ def get_set_cell_value_code(new_value, previous_value, zero_next_cell_if_necessa
     def get_abc(offset):
         # returns a,b,c such that a*b+c=offset and a+b+c is minimal
 
-        min_a, min_b, min_c = offset, 1, 0
-        min_sum = min_a + min_b + min_c
+        min_a, min_b, min_c = 0, 0, 0
```

Comment:
if offset = 0/1/2/3 then we won't run the loop and these will be zeros

Reply:
Will fix this. Even if naive wins and this never really happens.